logger/logger.go

```diff
@@ -43,7 +43,7 @@ func (c *Config) New(defaultOutput io.Writer) (*zap.Logger, error) {
 		encoder,
 		zapcore.Lock(zapcore.AddSync(w)),
 		c.Level,
-	)), nil
+	), zap.Fields(zap.String("cid", NextTraceID()))), nil
```

Comment:
What does `cid` stand for? Could we replace this with `trace_id` or at least `tid`?

Reply:
The PR request contains all the keys, including `cid`. I'd rather than that `correlation_id`

Reply:
Reading the PR request, the correlation id sounds more like a process id and using `pid` might be simpler since that's already an established abbreviation.

Reply:
It was explicitly changed from `pid` (the original choice) to a unique identifier, so that the same process identifier would not occur in the same set of log files _ever_.

Reply:
I think you misunderstood me. I didn't mean the physical process id.

Either way, is there something other than `cid`? I'm not particularly wed to a specific value, but `cid` doesn't follow any commonly known conventions. At the least, can we get a comment stating what it means in the code? The code itself says `NextTraceID()` and then assigns it to a value with `cid` so there's no mention of correlation in the code or comments.

I think I would be fine with that even if it isn't the ideal. I agree that correlation is a long word.

Any comments in this PR are not good enough documentation IMO. When someone is looking at a log file or looking at the code, they are not going to find this PR without some digging.

Reply:
For clarity, I've renamed `NextTraceID` as it is just an unique identifier. Also, will use `pid` vs `cid`